content/millennium/r4/financial/support.md

```diff
@@ -8,3 +8,4 @@ layout: api
 <%= render '/millennium/r4/financial/support.*' %>
 
 * [Coverage](/millennium/r4/financial/support/coverage)
+* [InsurancePlan](/millennium/r4/financial/support/insurance-plan)
```

Comment:
Need to change this as per Bhavya's comment above.

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/809/commits/f285387f72f531a780abd964a8c1b3335e9d5385#diff-1b2e7eaf4e7886cd7feb2c3259aaf69b731e611ada9084e5910d2e63deff97ddR13